pkg/utilities/util.go

```diff
@@ -0,0 +1,130 @@
+package util
+
+import (
+	"bytes"
+	"fmt"
+	"html/template"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	serializer "k8s.io/apimachinery/pkg/runtime/serializer/yaml"
+	"k8s.io/client-go/discovery"
+	"k8s.io/client-go/discovery/cached/memory"
+	"k8s.io/client-go/restmapper"
+)
+
+type TemplateArguments struct {
+	ClusterName        string
+	KeyPairName        string
+	AmiID              string
+	NodeRole           string
+	NodeRoleArn        string
+	NodeSecurityGroups []string
+	Subnets            []string
+}
+
+func NewTemplateArguments() *TemplateArguments {
```

Comment:
We might need to rethink this part.
It allows templating YAML files with env variables, but I think we should be more generic here, otherwise these variables don't mean much for Kubedog

Reply:
Based on our slack conversation, I will remove the templates.